extensions/http/jersey/src/main/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/UnexpectedExceptionMapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - Initial implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.extension.jersey.mapper;
+
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.ExceptionMapper;
+
+import java.util.Map;
+
+import static jakarta.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static jakarta.ws.rs.core.Response.Status.NOT_IMPLEMENTED;
+
+/**
+ * Exception mapper that catches all the exceptions that are not handled by the other mappers, mainly:
+ * - java.lang.*Exception
```

Comment:
actually subclasses of `java.lang.Throwable` according to the class' type definition

Reply:
the description was meant to say, all the exceptions belonging to `java.lang` package, because not all the subclasses of `Throwable` (that are, in fact, all the possible exceptions) are handled here, as some are handled by other mappers